commands/version/index.js

```diff
@@ -43,7 +43,15 @@ class VersionCommand extends Command {
     return ["publish"];
   }
 
-  initialize() {
+  get requiresGit() {
+    return (
+      this.commitAndTag || this.pushToRemote || this.options.allowBranch || this.options.conventionalCommits
+    );
+  }
+
+  configureProperties() {
+    super.configureProperties();
```

Comment:
I must admit I had a bad reaction when I first saw this `super` call, but now... I really like it! It does a good job of compartmentalizing the operations, as you'd expect a class method to do.

Reply:
I wasn't 100% sure about this either, but it seemed legit :)